validator/src/main.rs

```diff
@@ -977,6 +978,18 @@ pub fn main() {
         .pop()
         .unwrap();
 
+    let accounts_hash_cache_path = matches
+        .value_of("accounts_hash_cache_path")
+        .map(Into::into)
+        .unwrap_or_else(|| ledger_path.join(AccountsDb::DEFAULT_ACCOUNTS_HASH_CACHE_DIR));
```

Comment:
the help makes no mention of a default

Reply:
since we can't resolve the path at help generation time, something like...
```rust
format!("[default: LEDGER_DIR/{}]", AccountsDb::DEFAULT_ACCOUNTS_HASH_CACHE_DIR)
```
at the end of the help message